.github/workflows/snyk_security.yaml

```diff
@@ -0,0 +1,21 @@
+name: Snyk Security
+
+on: [ push, pull_request ]
+
+jobs:
+  snyk:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Test for open source vulnerabilities and license issues.
+        uses: snyk/actions/python-3.8@master
```

Comment:
I guess we don't care about this action being python-3.8 since this wouldn't be invoking elliott in any way. This can be the latest version of python / what the action supports

Reply:
Or am I confusing this.. does it include 3.8 relevant things in scan maybe?

Reply:
No, I don't think it matters. Tested it on my art-bot fork. Its working.

Reply:
https://github.com/snyk/actions/blob/806182742461562b67788a64410098c9d9b96adb/python-3.8/action.yml#L22 looks to be the important part that we want - basically communicate to snyk to look for 3.8 relevant vulnerabilities

Reply:
Ahh I see. Okay I'll add that change in.

Reply:
Things would break with python version above 3.8 - I don't know if pip would break. I am good with using the 3.8 action

Reply:
I did a test run and it didn't break. But since 3.8 is our production env, its better that we keep it that way I guess